content/docs/command-reference/repro.md

```diff
@@ -182,7 +182,8 @@ up-to-date and only execute the final stage.
 - `--pull` - [pulls](/doc/command-reference/pull) dependencies and outputs
   involved in the stages being reproduced, if they are found in the
   [default remote storage](/doc/command-reference/remote/default). Note that it
-  tries the local run-cache first (unless `--no-run-cache` is also used).
+  includes pulling any available run-cache (unless `--no-run-cache` is also
+  used).
```

Comment:
I'm not sure that this new text is correct but the previous one was unnecessary (already explained earlier that `repro` tries the run-cache first, which is not related to `--pull`).

Reply:
> not sure that this new text is correct

Hi @efiop maybe you can confirm about this please? Thanks

Reply:
Cc @pmrowla maybe you know? Tx

Reply:
`repro --pull` pulls outputs for pipeline stages which have been restored via the run-cache. We should probably note that it does not pull data dependencies (see https://github.com/iterative/dvc/issues/4742)

Reply:
Hmmm ok thanks @pmrowla I'm going to rewrite this whole option desc... Resolving this!